### PR TITLE
Missing images for local CSS file (jquery)

### LINKED ISSUE #12715: Missing images for local CSS file
### Description

Local CSS file [jquery.dataTables.css](https://github.com/astropy/astropy/blob/main/astropy/extern/jquery/data/css/jquery.dataTables.css) refers to [several images](https://github.com/astropy/astropy/blob/main/astropy/extern/jquery/data/css/jquery.dataTables.css#L49). They are the little arrows next to each sortable column. These images exist at the [CDN](http://cdn.datatables.net/), but they are missing in the repo. This makes that arrows are not displayed when using local files. 

### Expected behavior

image

### Actual behavior

image

### Steps to Reproduce

```python
from astropy.table import Table 
t = Table([[1, 2, 3], [4, 5, 6]], names=('a', 'b')) 
t.show_in_browser(jsviewer=True)
```

### System Details

```
>>> import platform; print(platform.platform())
Linux-5.10.76-linuxkit-x86_64-with-glibc2.2.5
>>> import sys; print("Python", sys.version)
Python 3.8.12 (default, Dec 21 2021, 10:45:09) 
[GCC 10.2.1 20210110]
>>> import numpy; print("Numpy", numpy.__version__)
Numpy 1.22.0
>>> import erfa; print("pyerfa", erfa.__version__)
pyerfa 2.0.0.1
>>> import astropy; print("astropy", astropy.__version__)
astropy 5.0
>>> import scipy; print("Scipy", scipy.__version__)
Scipy 1.7.3
>>> import matplotlib; print("Matplotlib", matplotlib.__version__)
Matplotlib 3.5.1
```

### PR DESCRIPTION
**Pull request by vilardellsalles**
### Description

This pull request is to include missing images required by the CSS file. 

Fixes #12715.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Do the proposed changes actually accomplish desired goals?
- [x] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [x] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [x] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [x] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [x] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label.
- [x] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [x] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [x] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

File: docs/changes/extern/12716.bugfix.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Images added from 'https://cdn.datatables.net/1.10.12/images/'
```
Comment (pllim) [inline docs/changes/extern/12716.bugfix.rst:1]: Don't think quotes are necessary. Also, would be nice to concisely explain why they were added. Thanks!
```suggestion
Images added from https://cdn.datatables.net/1.10.12/images/
```

File: docs/changes/extern/12716.bugfix.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Local CSS file `jquery.dataTables.css` refers to several images called
+`sort*.png`. They are the little arrows next to each sortable column. These
+images exist at the CDN (https://cdn.datatables.net/1.10.12/images/), but they
+were missing in this repo. Images required to display the corresponding arrows
+were added.
```
Comment (taldcroft) [inline docs/changes/extern/12716.bugfix.rst:5]: The bugfix entry should focus on the user perspective, not developer details. I'd suggest below. BTW does this apply to `show_in_notebook` as well?

```suggestion
Fixed a bug in ``Table.show_in_browser`` using the ``jsviewer=True`` option
to display the table with sortable columns. Previously the sort direction arrows
were not being shown due to missing image files for the arrows.
```

The useful information originally in the change log should probably go in the PR description.

**Review by taldcroft — CHANGES_REQUESTED** [review]
I tested this out and confirmed that it fixes the problems with arrows not being shown. So looks good but the change log entry should be updated.

**Comment by taldcroft** [comment]
I also just noticed that the change log file is in `extern`. Technically that is correct, but I'd say to put it into the `table` subpackage since it will get better visibility and that is the user-facing impact of this bugfix.

File: docs/changes/extern/12716.bugfix.rst (hunk 2 above)
Comment (vilardellsalles) [reply docs/changes/extern/12716.bugfix.rst]: >. BTW does this apply to `show_in_notebook` as well?

I don't think so. `show_in_notebook` uses the `JSViewer` class with the default `use_local_files=False`. Local files are used when `use_local_files=True`, like  in `show_in_browser`.

**Review by taldcroft — APPROVED** [review]
Looks good, thanks!

**Pull request merged**